components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -102,19 +103,19 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
                         }
                     },
                     new ExecutionDataflowBlockOptions {
-                        CancellationToken = cancellationToken
+                        CancellationToken = cancellationToken,
```

Comment:
>, [](start = 61, length = 1)

please remove the comma if there is only one parameter used

Reply:
Many people (incl. me) consider this a [best-practice](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8). There are multiple discussions about this on the Internet and C#, JavaScript and other languages have support for it and there is a reason for that.